.gitignore

```diff
@@ -213,3 +213,9 @@ new.json
 # Keys
 *.pem
 .ok.sql
+
+# direnv/envrc cache
+.direnv
+
+# Nix build output
+result
```

Comment:
```suggestion
result*
```

Reply:
why the star?

Reply:
`nix build .#spacetimedb-standalone .#spacetimedb-cli nixpkgs#sometool` will produce `./result`, `./result-1` and `./result-2`.

Reply:
TIL!